src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.scss

```diff
@@ -0,0 +1,4 @@
+.col-3 .col-4 .col-5 {
```

Comment:
You are overriding boostrap classes. I think it's better create a new class.

Reply:
Can you give me more information?

Reply:
It is ok. As far as I know, there are bootstrap classes named as `col-3` `col-4` `col-5`. So, I think is not a good idea to re-define them, that could cause unexpected behaviours. 

It is better to define a new class. e.g :
```
.myNewClass {
  padding: 1rem 0rem 2rem;
  overflow: hidden;
}
```

Reply:
If I am wrong somehow, please let me know.